config/environments/test.rb

```diff
@@ -56,6 +56,10 @@
   config.gias_api_user = "gias-user"
   config.gias_api_password = "gias-password"
 
+  config.zendesk_url = "https://www.example.com"
+  config.zendesk_username = "zendesk-username"
+  config.zendesk_token = "zendesk-token"
```

Comment:
Wouldn't it be better to move these envars into the encrypted rails credentials files?
Similarly to what we do with the credentials to DQT api.

Reply:
See chat in Slack